marlowe-dashboard-client/src/Capability/Marlowe.purs

```diff
@@ -0,0 +1,78 @@
+module Capability.Marlowe
+  ( class MonadMarlowe
+  , marloweCreateWalletCompanionContract
+  --, marloweCreateContract
+  , marloweApplyInputs
+  , marloweWait
+  , marloweAuto
+  , marloweRedeem
+  ) where
+
+import Prelude
+import AppM (AppM)
+import Bridge (toBack)
+import Capability.Contract (class MonadContract, activateContract, invokeEndpoint)
+import Capability.ContractExe (marloweContractExe, walletCompanionContractExe)
+import Control.Monad.Except (lift)
+import Data.Map (Map)
+import Data.RawJson (RawJson(..))
+import Data.Tuple.Nested ((/\))
+import Foreign.Generic (encode)
+import Foreign.JSON (unsafeStringify)
+import Halogen (HalogenM)
+import MainFrame.Types (WebData)
+import Marlowe.Semantics (Contract, Input, Party, Slot)
+import Plutus.PAB.Webserver.Types (ContractActivationArgs(..))
+import Plutus.V1.Ledger.Crypto (PubKeyHash)
+import Plutus.V1.Ledger.Value (TokenName)
+import Types (ContractInstanceId, MarloweParams)
+import WalletData.Types (Wallet)
+
+-- The `MonadMarlowe` class provides a window on the `MonadContract` class with function
+-- calls specific to the Marlowe Plutus contract.
+class
+  MonadContract m <= MonadMarlowe m where
```

Comment:
Same comment about the prefix, and also a little confused on what `MonadMarlowe` or `ManageMarlowe` would mean... Marlowe is implicit in everything we do in this app, when we talk about a Contract is a Marlowe contract. I think some of the functions here should reside in the Contract API (wait, applyInputs) and others should move to a Wallet or WalletCompanion API (redeem, createWalletCompanion)

Reply:
as above, this is the capability I envisage us using here - `Capability.Contract` is lower level and will eventually be used by the new pab-client